test/render-utils.test.js

```diff
@@ -0,0 +1,22 @@
+import { renderItem } from '../render-utils.js';
```

Comment:
Nice job Cole, its excellent that you're getting comfortable with testing. This will come in handy throughout the rest of the course.